hugegraph-test/src/main/java/com/baidu/hugegraph/core/AuthTest.java

```diff
@@ -1270,6 +1273,149 @@ public void testRolePermission() {
         Assert.assertEquals(expected, role.toJson());
     }
 
+    @Test
+    public void testCreateProject() {
+        HugeGraph graph = graph();
+        HugeProject project = makeProject("test_project",
+                                          "this is a test project");
+        Id id = graph.authManager().createProject(project);
+        Assert.assertNotNull(id);
+        project = graph.authManager().getProject(id);
+        Assert.assertNotNull(project);
+        Assert.assertEquals("this is a test project", project.desc());
+        Assert.assertEquals("test_project", project.name());
+        Assert.assertTrue(!Strings.isNullOrEmpty(project.adminGroupId()));
+        Assert.assertTrue(!Strings.isNullOrEmpty(project.opGroupId()));
+        Assert.assertTrue(!Strings.isNullOrEmpty(project.targetId()));
+
+        //Check name is unique index
+        HugeProject sameNameProject = makeProject("test_project",
+                                                  "this is a test "
+                                                  + "project another");
+        Assert.assertThrows(HugeException.class,
+                            () -> {
+                                graph.authManager()
+                                     .createProject(sameNameProject);
+                            });
+    }
+
+    @Test
+    public void testDelProject() {
+        HugeProject project = makeProject("test_project", null);
+        AuthManager authManager = graph().authManager();
+        Id projectId = authManager.createProject(project);
+        Assert.assertNotNull(projectId);
+        HugeProject deletedProject = authManager.deleteProject(projectId);
+        Assert.assertThrows(NotFoundException.class,
+                            () -> {
```

Comment:
move to the previous line

Reply:
this line so long that dont place all, so i move it to next line;

Reply:
please refer to the style of others test cases.